instrumentation/log4j/log4j-appender-1.2/javaagent/build.gradle.kts

```diff
@@ -9,6 +9,7 @@ muzzle {
     versions.set("[1.2,)")
     // version 1.2.15 has a bad dependency on javax.jms:jms:1.1 which was released as pom only
     skip("1.2.15")
```

Comment:
does it need to match line 11 at log4j/log4j-mdc-1.2/javaagent/build.gradle.kts? comment will be helpful.

Reply:
> does it need to match line 11 at log4j/log4j-mdc-1.2/javaagent/build.gradle.kts? comment will be helpful.

Because in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/2986b570b42c5fa2f5c0901e7fd564f49eb065c1/instrumentation/log4j/log4j-appender-1.2/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/log4j/appender/v1_2/LogEventMapper.java#L118, it used MDC that is contained in 1.2+, so it will not have same problem with log4j-mdc-1.2 here.